base/server.go

```diff
@@ -39,11 +39,19 @@ type Server struct {
 	multiDBDSN   string
 	multi        *multi
 
-	runOptions kbchat.RunOptions
+	runOptions    kbchat.RunOptions
+	serverOptions ServerOptions
 }
 
-func NewServer(name, announcement string, awsOpts *AWSOptions, multiDBDSN string, runOptions kbchat.RunOptions) *Server {
-	return &Server{
+type ServerOptions struct {
+	ReadMessagesFromSelf bool
```

Comment:
can you put a comment here that it's helpful to reverse this bool in dev? otherwise you need two services running to talk to the bot.

Reply:
Sure! I usually have two services running 😅